packages/xo-server/src/xo-mixins/plugins.js

```diff
@@ -60,7 +60,9 @@ export default class {
     const plugin = (this._plugins[id] = {
       configurationPresets,
       configurationSchema,
-      configured: !configurationSchema,
+      configured:
```

Comment:
@julien-f should we considerate a plugin with optional configuration configured or should we instead of this implementation make it loadable on loading the plugin.

Reply:
As per current implementation, a plugin is loadable if it's configured.

And configured means that the plugin has a configuration valid as per its configuration schema.

Therefore being configured and having a valid empty configuration means the same thing.